conf_linux_public.py

```diff
@@ -107,13 +107,15 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
        cmd=print_gcc_version(GCC_LATEST, ENABLE_DEVTOOLSET),
        verbose=True)
 
-cmake_command = ['cmake',
-                 '--no-warn-unused-cli',
-                 '-Wno-dev -G "Unix Makefiles"',
-                ]
-
-cmake_command.append('-DCMAKE_C_FLAGS_RELEASE="-O2 -Wformat -Wformat-security -Wall -Werror -D_FORTIFY_SOURCE=2 -fstack-protector-strong"')
-cmake_command.append('-DCMAKE_CXX_FLAGS_RELEASE="-O2 -Wformat -Wformat-security -Wall -Werror -D_FORTIFY_SOURCE=2 -fstack-protector-strong"')
+cmake_command = ['cmake']
+if args.get('compiler') == "clang":
+    cmake_command.append('-DCMAKE_C_COMPILER=clang-6.0')
+    cmake_command.append('-DCMAKE_CXX_COMPILER=clang++-6.0')
+else:
```

Comment:
Use `elif args.get('compiler') == "gcc":` but need to change `config.py` to use initialize `compiler` but not `compiler_version`. Hm, but we will lose "default" behavior...